### ISSUE TITLE
Infra: Setup CI via github actions or TravisCI

### ISSUE THREAD

**Issue by giorgiozoppi** [openMF/digital-bank-ui#6]
This repository's issues are reserved for feature requests and bug reports.

#### Feature Request
We need a CI installed for any commit. Each commit will build the app and run tests.

**Comment by edcable** [comment]
@giorgiozoppi Can we use Github Actions for this?

**Comment by giorgiozoppi** [comment]
Yes, but until 10 i cannot use my account in amazon due credit card loss.
